lighttpd/tox.ini

```diff
@@ -5,6 +5,8 @@ envlist =
     py{27,37}-latest
 
 [testenv]
+description =
```

Comment:
What's this?

Reply:
It's the trick Ofek used to define environments where we're going to run e2e tests. As we don't control (yet) which python version the agent is running, we're doing it explicitly here.